app-android-xamarin/Moodis/Moodis/Resources/values/strings.xml

```diff
@@ -37,6 +37,7 @@
   <string name="stronger_password">Password must be stronger</string>
   <string name="user_created">User created successfully</string>
   <string name="user_exists">User already exists</string>
+  <string name="user_face_exists">A user with such a face already exists try to sign in.</string>
```

Comment:
Just a suggestion, but maybe this string should be: `You already have an account registered in Moodis`